modules/components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SocialUtil.java

```diff
@@ -44,6 +45,18 @@ public static String getTenantDomain() {
 
 	}
 
+	public static String getTenantDomainFromTenantId(int tenantId) {
```

Comment:
This method already exist in Carbon UserRelm, don't write your own method. this one is inefficient.